redirects/nginx.conf

```diff
@@ -84,4 +84,13 @@ http {
 
     return 301 https://{{ env "TARGET_DOMAIN" }}$request_uri;
   }
+
+  server {
+    listen {{port}};
+    server_name dashboard.data.gov;
+
+    location /validate {
+        return 301 https://{{ env "TARGET_DOMAIN" }}/dcat-us/validator;
```

Comment:
I hadn't realized this earlier.. but this won't work because the `TARGET_DOMAIN` is `data.gov` so this is redirecting 
- `dashboard.data.gov` to `data.gov/dcat-us-validator`

I think @robert-bryson's point about adding this redirect into the catalog-proxy is more valid.  But it complicates things because `dashboard.data.gov` would need to be tied to `catalog-proxy` app which seems weird to me.

Reply:
is this just an env variable? can we not make a redirects app for dashboard urls and then put everything else catalog into `catalog-proxy`?

Reply:
We could use a different variable, but we need to add that.

Reply:
@robert-bryson do we need this change?